proto/viam/app/datapipelines/v1/data_pipelines.proto

```diff
@@ -29,8 +29,8 @@ service DataPipelinesService {
   // DisableDataPipeline disables a data pipeline.
   rpc DisableDataPipeline(DisableDataPipelineRequest) returns (DisableDataPipelineResponse);
 
-  // ListPipelineRuns lists the runs of a data pipeline.
-  rpc ListPipelineRuns(ListPipelineRunsRequest) returns (ListPipelineRunsResponse);
+  // ListDataPipelineRuns lists the runs of a data pipeline.
+  rpc ListDataPipelineRuns(ListDataPipelineRunsRequest) returns (ListDataPipelineRunsResponse);
```

Comment:
Also we're in the `datapipelines` package -- the stylistically correct name would be `ListRuns`. 

Using `datapipelines.ListDataPipelineRuns` is inferior to `datapipelines.ListRuns`.

Reply:
this one is true for Go but not necessarily for gRPC -- in fact the proto package name here is `viam.app.datapipelines.v1` and the AIP examples are all `ListBooks` `GetBook` etc: https://google.aip.dev/132

it's also the existing pattern in Viam APIs: https://github.com/viamrobotics/api/blob/main/proto/viam/app/dataset/v1/dataset.proto https://github.com/viamrobotics/api/blob/main/proto/viam/app/mltraining/v1/ml_training.proto

Reply:
Strictly speaking, yes. But almost all of the client and server architecture is Go. It's also a preferred style in many other languages (e.g: python and Java) that are simply too dated to have an official style guide on the relationship between package and type names.